test/desktop/features/handled_errors.feature

```diff
@@ -25,7 +25,12 @@ Feature: Handled Errors and Exceptions
         And the event "device.runtimeVersions.unityScriptingBackend" is not null
         And the event "device.runtimeVersions.dotnetScriptingRuntime" is not null
         And the event "device.runtimeVersions.dotnetApiCompatibility" is not null
-        And the event "app.type" equals "Mac OS"
+        And the event "app.type" equals the platform-dependent string:
+        | macos | "Mac OS" |
```

Comment:
Needs to be without the `"`s.  Also lose the blank lines below.

Reply:
Also, while I'm being pedantic, please indent line 29/30 to align with other similar steps.

Reply:
done